test/unit/lifecycle/properties.js

```diff
@@ -25,7 +26,11 @@ describe('lifecycle/property', () => {
   });
 
   describe('props declared as attributes with ES2015 classes are linked', () => {
+    const skip = !classStaticsInheritance();
```

Comment:
Not saying it won't work. Just surprised that it compiles ES6 code to ES6. Don't know how it can increase browser support.